_data/users.yml

```diff
@@ -81,6 +102,15 @@
     consistently fast performance at any scale and budget.
   urltext:
   url: https://resurface.io/
+- name: Salesforce
+  logo: /assets/images/logos/salesforce.jpg
+  # https://engineering.salesforce.com/big-data-big-decisions-690a00fad88d
+- name: Sentinel One
+  logo: /assets/images/logos/sentinelone.png
+  # https://sentinelone-tech.medium.com/how-sentinelone-optimizes-big-data-at-scale-b957b5fa7b8a
+- name: Shopify
+  logo: /assets/images/logos/shopify.png
+  # https://shopify.engineering/shopifys-data-science-engineering-foundations
```

Comment:
Are these just placeholders or we're planning on using these blogs as testimonials? I'm speaking right now with Nick from Shopify and he's getting clearance from his boss. Should we wait to get a green light before we merge this?